client/src/main/java/gui/playing/PlayBoardController.java

```diff
@@ -102,6 +117,8 @@ private void populateSeaWithSeaFields() {
           field.marked();
           lastField = field;
         });
+
```

Comment:
empty line

Reply:
I think it's more readable with empty line

Reply:
👍